includes/polling/sensors/runtime/apc.inc.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2016 Cercel Valentin <crc@nuamchefazi.ro>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+$sensor_value = snmp_get($device, $sensor['sensor_oid'], '-OUqnvt');
```

Comment:
I'd suggest changing the generic code and adding `-Ot`  there is no reason we would want a human readable format for any time based sensor.